geo/src/utils.rs

```diff
@@ -127,6 +127,25 @@ where
     }
 }
 
+/// Compute index of the lexicographically least point.
```

Comment:
For my own understanding: lexicographically as opposed to?

Reply:
Unlike real numbers, there is no de-facto ordering of points.  Lexicographic is to emphasize that we sort by x coord, and then break ties using y coord.

Reply:
> Unlike real numbers, there is no de-facto ordering of points. Lexicographic is to emphasize that we sort by x coord, and then break ties using y coord.

Could we make this a comment for future selves / maintainers?